### PR TITLE
Added 'Manage dashboard' tooltip, hides on click.

### PR DESCRIPTION
**Pull request by utkarshcmu**
Added "Manage dashboard" Tooltip for the settings icon. Also, this tooltip hides when we click on the icon.

Here is the demo:

Hovering over the settings icon shows the following tooltip:
screen shot 2015-09-18 at 12 55 43 am

Clicking on the settings icon removes the tooltip and shows the dropdown menu.
screen shot 2015-09-18 at 12 55 50 am

### CODE REVIEW

**Comment by utkarshcmu** [comment]
@torkelo How about this small feature?

File: .jshintrc [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "define": true,
     "require": true,
     "Chromath": false,
-    "setImmediate": true
+    "setImmediate": true,
+    "$": false
```
Comment (torkelo) [inline .jshintrc]: why did you add this property?

**Comment by torkelo** [comment]
nice :)

File: .jshintrc (hunk 1 above)
Comment (utkarshcmu) [inline .jshintrc]: Grunt run was complaining and threw the following error:

$ grunt
Running "jscs:src" (jscs) task
>> 155 files without code style errors.

Running "jshint:source" (jshint) task

public/app/features/dashboard/dashboardNavCtrl.js
  line 55  col 7  '$' is not defined.

✖ 1 problem

Warning: Task "jshint:source" failed. Use --force to continue.

Aborted due to warnings.


Then I searched online a bit and found that we need to enable $ as a variable. Actually, this is the link which I followed: http://stackoverflow.com/questions/8852765/jshint-strict-mode-and-jquery-is-not-defined

**Comment by utkarshcmu** [comment]
One more commit to fix #2702.

File: .jshintrc (hunk 1 above)
Comment (torkelo) [inline .jshintrc]: well I think you misunderstood this :) the jshint error is CORRECTLY complaining that you are using an undefined variable. the correct thing would be to require jquery as a dependency in the top of the js file, not to remove the validation test that checks for undefined variables!

Comment (torkelo) [inline .jshintrc]: if your still unsure about what I mean, please google requirejs or look at some directive file in grafana that is also using jquery , and remove the change to jshintrc

**Comment by utkarshcmu** [comment]
@torkelo Code looks better now, or is there anything which we should change?

**Comment by torkelo** [comment]
merged into master :) Thanks for contributing to Grafana!

**Comment by utkarshcmu** [comment]
@torkelo Thanks for optimizing my code. I am still learning angularjs/jquery and its great to learn from you. :)
